src/github-patcher.css

```diff
@@ -11,11 +11,11 @@ a[href$='/forks'] strong,
 /* Third party stats. "GitHub Readme Stats" */
 /* NOTE: Do not hide other endpoints like `api/top-langs` */
 a:has(> img[data-canonical-src^='https://github-readme-stats.vercel.app/api?']) {
-  display: none;
+  visibility: hidden;
```

Comment:
If we can use `display: none` likely current implementation, it seems simple and reasonable behavior to me.
So please use `display: none` again. And scripting.insertCSS will use same CSS. Eventually you should consider adjustment it in scripting.removeCSS

Reply:
No. You didn't consider `Eventually you should consider adjustment it in scripting.removeCSS`

Current implementation cannot disabling behavior even if changing the slider to off